src/Components/Context/AuthContext/AuthContext.tsx

```diff
@@ -0,0 +1,42 @@
+import { jwtDecode } from "jwt-decode";
+import { createContext, useEffect, useState } from "react";
+
+export let AuthContext = createContext(null);
+export default function AuthContextProvider(props) {
+  const [userRole, setUserRole] = useState(null);
+  // ******************* to baseUrl for apis *******************
+  const baseUrl = `http://upskilling-egypt.com:3000/api/v0`;
+  const requstHeaders = {
+    Authorization: `Bearer ${localStorage.getItem("adminToken")}`,
```

Comment:
You 'll need to remove `Bearer` here as the token coming from the backend is `Bearer TokenValue`, so if you used the current token the request header will be `Bearer Bearer TokenValue`